### ISSUE TITLE
Add ModCorral style feature?

### ISSUE THREAD

**Issue by originalfoo** [arislancrescent/CS-SkylinesOverwatch#3]
Something like this might be useful component of Overwatch...

http://steamcommunity.com/sharedfiles/filedetails/?id=419090722

ISTR there's also a menu accessed by hotkey that some mods use. Would be good to centralise and streamline

**Comment by arislancrescent** [comment]
I played around with that mod when it first came out. It seems like a good idea. But I'm wondering if it would be better to have a separate UI modding library? Either way, once I get all my AI stuff done, I might take a stab at incorporating UI into Skylines Overwatch.

p.s. Now that T++ and TM can work together, maybe you guys can implement that NetLane variable I mentioned? CO pretty much ignored me when I asked about it during their reddit AMA. So, if you guys can put it in, then I'll see if my car pathing AI can ride off of your mods.

**Comment by originalfoo** [comment]
Yeah, the game front-end UI is currently terrible in every possible respect, a decent front-end UI API would be a huge win.

Yup, we'll see what we can do. It's prolly worth you adding an issue to https://github.com/joaofarias/csl-traffic to explain what is required and some use cases.
